.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    if: github.repository == 'hugovk/iniconfig'
```

Comment:
this one seems to be a oversight from your personal testing

Reply:
Oops, will fix!